spec/presenters/idv/in_person/ready_to_verify_presenter_spec.rb

```diff
@@ -120,6 +122,43 @@
     end
   end
 
+  describe '#partner_agency' do
+    subject(:partner_agency) { presenter.partner_agency }
+
+    it 'returns partner agency' do
+      expect(partner_agency).to eq('http://localhost:3000')
+    end
+  end
+
+  describe '#days_reminder' do
+    subject(:days_reminder) { presenter.days_reminder }
+
+    context '4 days until due date' do
+
+      before do
```

Comment:
looking for feedback on how to test this days_reminder. Need enrollment.days_to_due_date to have a mock value but it fails b/c:
`NoMethodError:
       undefined method `days_to_due_date' for :enrollment:Symbol`

Reply:
Can we initialize the enrollment (on line 16) such that it produces the days we're expecting?

Otherwise we could stub the enrollment like...

```
allow(enrollment).to receive(:days_to_due_date).and_return(4)
```